Source/WebKit/UIProcess/API/APIContentRuleListStore.cpp

```diff
@@ -81,17 +81,15 @@ ContentRuleListStore::ContentRuleListStore(const WTF::String& storePath)
 
 ContentRuleListStore::~ContentRuleListStore() = default;
 
-// FIXME: Remove legacyFilename in 2022 or 2023 after users have had a chance to run the updating logic.
-static ASCIILiteral constructedPathPrefix(bool legacyFilename)
+static constexpr ASCIILiteral constructedPathPrefix()
```

Comment:
Nit: Since you are changing all the call sites - you probably could change the name of this method because nothing is really being constructed anymore.